roomserver/internal/input/input_latest_events.go

```diff
@@ -303,8 +307,14 @@ func (u *latestEventsUpdater) makeOutputNewRoomEvent() (*api.OutputEvent, error)
 		latestEventIDs[i] = u.latest[i].EventID
 	}
 
+	var outputType api.OutputRoomEventType
+	if u.isHistorical {
+		outputType = api.OutputRoomState
```

Comment:
`isHistorical` feels like the wrong naming here - presumably you could get historical message events which then confusingly get marked as `OutputRoomState`?